test/font_test.py

```diff
@@ -596,7 +594,7 @@ def query(self, bold=False, italic=False, underline=False, antialiase=False):
                 modes.append("underlined")
             if antialiase:
                 modes.append("antialiased")
-            text = "%s (y/n):" % ("-".join(modes),)
+            text = "{} (y/n):".format("-".join(modes))
```

Comment:
this can be further simplified manually to something like `f"{'-'.join(modes)} (y/n):"`

Reply:
```suggestion
            text = f"{'-'.join(modes)} (y/n):"
```